src/main/java/com/example/bookstore/model/Role.java

```diff
@@ -0,0 +1,26 @@
+package com.example.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class Role {
```

Comment:
Did you test your solution? Does it work without getters and setters? Also, Table annotation missed